project_task_pull_request_state/models/project_task_state.py

```diff
@@ -0,0 +1,15 @@
+from odoo import fields, models
+
+
+class ProjectTaskState(models.Model):
+    _inherit = "project.task"
+
+    pr_state = fields.Selection([
```

Comment:
Better use function to define variables because it will be easier to override values later if needed.
See example[ here](https://github.com/OCA/stock-logistics-warehouse/blob/5f576f26eaa110a7f015c3c14a135a2fb185e846/stock_location_product_restriction/models/stock_location.py#L13)